packages/controller/src/resources/loki/index.ts

```diff
@@ -833,7 +862,7 @@ export function LokiResources(
                 accessModes: ["ReadWriteOnce"],
                 resources: {
                   requests: {
-                    storage: "10Gi"
+                    storage: "30Gi"
```

Comment:
Given the load we have seen thus far this change is not necessary and because it increases the cost profile we should not do it now. Or do you prefer to keep the sizes in sync with Cortex?

Reply:
Ok I reset all the Loki PVCs back to 10Gi